apps/api/src/api.module.ts

```diff
@@ -2,14 +2,14 @@ import { Module } from '@nestjs/common';
 import { EventEmitterModule } from '@nestjs/event-emitter';
 import { BullModule } from '@nestjs/bullmq';
 import { ScheduleModule } from '@nestjs/schedule';
-import { RedisModule } from '@liaoliaots/nestjs-redis';
+import { RedisModule } from '@songkeys/nestjs-redis';
```

Comment:
This was erroring as the other flavor of redis is not upto date, changing to this similar one compatible with newer nestjs, semantics around redis remains the same